scripts/abi_artifact_generator/templates/setup_arguments.S.jinja_template

```diff
@@ -0,0 +1,53 @@
+/*#-
```

Comment:
Can you pass me the set of generated files?
Looking at the generated file and tracking it back to the template is much more effective to get gist of what we're doing.

NOTETOSELF: review the templates.

Reply:
No need, I'll build locally. It's simpler.

Reply:
Closing this then.